src/main/java/pacman/Main.java

```diff
@@ -4,30 +4,18 @@
 class Main extends Frame{
 
     //attrib
-    private pacman p;
-    private food f;
-    private ghost g;
-    private ArrayList <wall> w; 
-
+    private Pacman objectPacman;
+    private Dots objectDots;
+    private Ghost objectGhost;
```

Comment:
object as variable name could be redundant, please remove "object" from variable name

Reply:
ok